lib/options.js

```diff
@@ -7,7 +7,10 @@ const isDev = () => process.env.NODE_ENV === 'development';
 const THIRTY_DAYS = 30 * 24 * 60 * 60 * 1000;
 
 const DEFAULT_OPTIONS = {
-    databaseUrl: process.env.DATABASE_URL,
+    databaseUrl:
+        process.env.DATABASE_URL || process.env.DATABASE_HOST
+            ? defaultDatabaseUrl()
```

Comment:
default should only be set up in local dev. We might want to move this logic out to package.json

Reply:
default option database_url only be setting up when DATABASE_URL || DATABASE_HOST is specificated, otherwise it will be null.